.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I would move all these rules into the eslint config.... prettier is integrated into eslint.